conn.go

```diff
@@ -54,6 +54,7 @@ type watchPathType struct {
 	wType watchType
 }
 
+// Dailer is the connection dialer function for ZK.
```

Comment:
Spelling mistake.  Should be `Dialer` not `Dailer`

Reply:
This is sorta redundant (i.e Dialer is a dialer).

Perhaps something like:

```
// Dialer is a function to be used to establish a connection to a single host.
```

Reply:
Yeah, that's better, I'v modified it.